### PR TITLE
Fix containerized merqury

### PR DESCRIPTION
**Pull request by mvdbeek**
Might be faster than https://github.com/bioconda/bioconda-recipes/pull/47201

FOR CONTRIBUTOR:
* [ ] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] License permits unrestricted use (educational + commercial)
* [ ] This PR adds a new tool or tool collection
* [ ] This PR updates an existing tool or tool collection
* [ ] This PR does something else (explain below)

### CODE REVIEW

**Comment by natefoo** [comment]
I'm +1 on not bumping the version since the previous version is broken with containers.

Also just for reference the coreutils bump is necessary since 8.32 was from defaults, does not exist in conda-forge, and no longer exists in defaults.

**Comment by bgruening** [comment]
I take care of the bioconda one

**Comment by natefoo** [comment]
In this case we can drop the addition of awk and sed requirements but will still need the coreutils change to build a new mulled container?

**Comment by bgruening** [comment]
Yes. Will do ... do you need it today or can we wait for the bot until tm?

**Comment by natefoo** [comment]
I think we can wait for the bot, thanks!

**Comment by mvdbeek** [comment]
> I'm +1 on not bumping the version since the previous version is broken with containers.

This makes it much harder to tell what version of a tool was run if users run into this on other servers. We should really always bump the version.

File: tools/merqury/test-data/output_02_qv.tabular [hunk 1]
```diff
@@ -1 +1 @@
-assembly	0	1037953		
+assembly	0	1037953	+inf	0
```
Comment (mvdbeek) [inline tools/merqury/test-data/output_02_qv.tabular:1]: It's normal that this changes, it just means that the bug is fixed. The problem is that the test data was wrong initially.

**Review by bgruening — APPROVED** [review]

**Pull request merged**